coreax/kernel.py

```diff
@@ -335,11 +335,17 @@ def divergence_x_grad_y_elementwise(self, x: ArrayLike, y: ArrayLike) -> Array:
 
 
 class SquaredExponentialKernel(Kernel):
-    """
+    r"""
     Define a squared exponential kernel.
 
-    :param length_scale: Kernel smoothing/bandwidth parameter
-    :param output_scale: Kernel normalisation constant
+    Given :math:`\lambda =`'length_scale' and :math:`\rho =`'output_scale', the squared
+    exponential kernel is defined as
+    :math:`k: \mathbb{R}^d\times \mathbb{R}^d \to \mathbb{R}`,
+    :math:`k(x, y) = \rho \exp(\frac{||x-y||^2}{2 \lambda^2})` where
```

Comment:
Renders more nicely in the built docs
```suggestion
    :math:`k(x, y) = \rho * \exp(\frac{||x-y||^2}{2 \lambda^2})` where
```